win32/ctags_vs2013.vcxproj

```diff
@@ -166,7 +167,7 @@
     <ClCompile Include="..\optlib\qemuhx.c" />
     <ClCompile Include="..\optlib\scss.c" />
     <ClCompile Include="..\optlib\systemtap.c" />
-    <ClCompile Include="..\parsers\abaqus.c" />                       
+    <ClCompile Include="..\parsers\abaqus.c" />

```

Comment:
If possible, I would like you to create a separate commit for this change because this change is nothing to do with kotlin parser.
The commit log for this change may be "buildsys: trim whitespace from ctags_vs2013.vcxproj".

Reply:
Sorry, this was an automated fix done by my Geany configuration, which I overlooked. I'll move it to separate commit (or remove altogether, if you prefer).

Reply:
Removed